site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -257,48 +257,107 @@ export const CreateWorkspacePageView: FC<
             </div>
           )}
 
-          {/* Rich parameters */}
-          {props.templateParameters && props.templateParameters.length > 0 && (
-            <div className={styles.formSection}>
-              <div className={styles.formSectionInfo}>
-                <h2 className={styles.formSectionInfoTitle}>
-                  Rich template params
-                </h2>
-                <p className={styles.formSectionInfoDescription}>
-                  Those values are provided by your template&lsquo;s Terraform
-                  configuration.
-                </p>
+          {/* Immutable rich parameters */}
+          {props.templateParameters &&
+            props.templateParameters.filter((p) => !p.mutable).length > 0 && (
+              <div className={styles.formSection}>
+                <div className={styles.formSectionInfo}>
+                  <h2 className={styles.formSectionInfoTitle}>
+                    Immutable parameters
+                  </h2>
+                  <p className={styles.formSectionInfoDescription}>
+                    Those values are provided by your template&lsquo;s Terraform
```

Comment:
Do we need to escape characters here? Or could that be automatically handled by react?

Reply:
That's a question to @BrunoQuaresma as I don't know if that's intended (copied from Parameters).

Reply:
What do you mean by escape characters?